### PR TITLE
Cryo tubes only ever send one radio message when finishing

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Cryo tubes will only ever send one message, even if they are auto ejecting.
/:cl:

Just put them on the same line, jesus.

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -179,10 +179,11 @@
 		on = FALSE
 		update_icon()
 		playsound(src, 'sound/machines/cryo_warning.ogg', volume) // Bug the doctors.
-		radio.talk_into(src, "Patient fully restored", radio_channel, get_spans(), get_default_language())
+		var/msg = "Patient fully restored."
 		if(autoeject) // Eject if configured.
-			radio.talk_into(src, "Auto ejecting patient now", radio_channel, get_spans(), get_default_language())
+			msg = "[msg] Auto ejecting patient now."
```
Comment (MrDoomBringer) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]:
```suggestion
			msg += " Auto ejecting patient now."
```

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm (hunk 1 above)
Comment (coiax) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: Does it matter

Comment (nicbn) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: + operator is faster, as [] operator formats the inner content

Comment (MrDoomBringer) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: also it follows convention better, we do it this way in the rest of our code

**Pull request merged**